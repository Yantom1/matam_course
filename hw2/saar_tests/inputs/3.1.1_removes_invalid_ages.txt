10000000, student, 121, 1899, 1
20000000, student, 120, 1900, 1
30000000, studentthree, 15, 2005, 1
50000000, studentfive, 16, 2004, 1
40000000, studentfour, -10, 2030, 1
60000000, studentsix, 0, 2020, 4
