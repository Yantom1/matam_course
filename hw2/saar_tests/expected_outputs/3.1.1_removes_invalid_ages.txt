20000000, student, 120, 1900, 1
50000000, studentfive, 16, 2004, 1
